package.json

```diff
@@ -0,0 +1,45 @@
+{
+  "name": "mv-react-bookstore",
+  "version": "0.1.0",
+  "private": true,
+  "dependencies": {
```

Comment:
Opps, you need to add `prop-types`

screenshot from pathwright;
![image](https://user-images.githubusercontent.com/31897434/83758107-6a18ab80-a669-11ea-85aa-1e6c5d8e5f0c.png)